Content.Shared/Posters/Components/PosterComponent.cs

```diff
@@ -0,0 +1,19 @@
+using Robust.Shared.Audio;
+using System.Threading;
+
+namespace Content.Shared.Posters.Components;
+
+[RegisterComponent]
+public sealed partial class PosterComponent : Component
+{
+    public CancellationTokenSource? CancelToken;
+
+    [DataField("foldedPrototype")]
+    public string? FoldedPrototype;
+
+    [DataField("removingTime")]
+    public float RemovingTime = 2.0f;
+
+    [DataField("removingSound")]
+    public SoundSpecifier? RemovingSound = new SoundPathSpecifier("/Audio/Effects/poster_removing.ogg");
+}
```

Comment:
1. Document your code
2. Datafield strings are obsolete
3. This needs predicting, it's not networked.
4. CancelTokens shouldn't be used for components.